pkg/bgp/extended-community.go

```diff
@@ -286,8 +286,10 @@ func type6(subType uint8, value []byte) string {
 		copy(l, value[3:])
 		s = fmt.Sprintf("%d:%d", value[0], binary.BigEndian.Uint32(l))
 	case 0x02:
+		fallthrough
+	case 0x03:
```

Comment:
Should it be 0x5?? I think in the previous commit it was 0x5 if I am not mistaken.

Reply:
Nope, it was. 0X03, EVPN Router’s MAC Extended Community | [RFC9135]